schema/bom-1.4.xsd

```diff
@@ -1186,11 +1186,11 @@ limitations under the License.
 
     <xs:complexType name="dependencyType">
         <xs:sequence minOccurs="0" maxOccurs="unbounded">
-            <xs:element name="dependency" type="bom:dependencyType"/>
+            <xs:element name="dependency" type="bom:refType"/>
```

Comment:
:x:  this is a breaking change. 

the following would no longer be valid, but was valid before

```xml
<bom nmlns="http://cyclonedx.org/schema/bom/1.4">
<dependencies>
  <dependency ref="foo">
    <dependencies>
     <dependency ref="bar" />
    </dependencies>
  </dependency>
</dependencies>
</bom>
```

Reply:
You are not wrong - this is a breaking change. Does this mean we have to wait until 1.5 to get this "aligned"?

Reply:
@stevespringett , @CycloneDX/core-team , @CycloneDX/industry-working-group